mbuild/tests/test_packing.py

```diff
@@ -41,7 +41,8 @@ def test_fill_region(self, h2o):
                                 region=[3, 2, 2, 4, 4, 3])
         assert filled.n_particles == 50 * 3
         assert filled.n_bonds == 50 * 2
-        assert np.min(filled.xyz[:,0]) >= 3
+        assert ((np.min(filled.xyz[:,0]) >= 3 - 0.01) or
+                (np.min(filled.xyz[:,0]) >= 3 + 0.01))
```

Comment:
I don't think these assertions are correct. Has the 0.01 been added as a tolerance buffer (this seems like a fairly large buffer)? If so, that should only be required on the lower end, i.e. I think this should read `assert np.min(filled.xyz[:,0]) >= 3 - 0.01`.

Reply:
Good catch, I mentioned in the issue #419 that the default tolerance for packmol is `0.01`. I could bump the tolerance to  `0.005` which may result in packing taking longer. I'll play with this a bit to see if I made a mistake with the assertion or if we are really running into an issue with the packmol tolerance.